client/src/components/Player.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Player = (props) => {
+  console.log("Player.js", props)
+    return (
+    <div className="cardContainer">
+        <div className="playerCard">
+        <h3>Player: {props.name}</h3>
+        <p>Country: {props.country}</p>
+        <p>Search Volume: {props.searches}</p>
+    </div>
```

Comment:
writing `props.` is fun right? wrong. DRY methods would ask us to do something like this

```
const Player = ({ name, country, searches }) => {
  console.log("Player.js", props)
    return (
    <div className="cardContainer">
        <div className="playerCard">
        <h3>Player: {name}</h3>
        <p>Country: {country}</p>
        <p>Search Volume: {searches}</p>
        </div>
    </div>
  );
}
```

by deconstructing props we save ourselves from writing `props.` a billion times in the document. This technique can be applied other places besides components.